scripts/python/command.py

```diff
@@ -0,0 +1,104 @@
+CONNECT = 1
```

Comment:
Why do we need this intermediate representations? Aren't command the strings themselves sufficient?

Reply:
This approach permits to modify command string without changing other portions of code.
It permits you to add aliases without touching anything else.
It's also less prone to typos.

The trade off is: keep the list updated.

Reply:
See https://github.com/nolash/weechat-pss/pull/11#discussion_r